queries.sql

```diff
@@ -7,3 +7,53 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered=TRUE;
 SELECT * from animals WHERE name <> 'Gabumon';
 SELECT * from animals WHERE weight_kg >=10.4 and weight_kg <= 17.3;
+BEGIN;
+update animals set species='unspecified' where species is null;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE species is NULL;
+COMMIT;
```

Comment:
- [ ] As stated above, please make sure to include queries for verifying that the change was made before and after committing